core/src/main/java/org/fao/geonet/kernel/search/IndexAndTaxonomy.java

```diff
@@ -44,5 +44,7 @@ public IndexAndTaxonomy(long version, GeonetworkMultiReader indexReader, Taxonom
     @Override
     public void close() throws IOException {
         indexReader.releaseToNRTManager();
+        //TODO: this has a taxonomyReader in it, and it should be "closed" (decrement a ref)
```

Comment:
The `taxonomyReader` is used in `LuceneSearcher` and `CatalogSearcher` during the searches to build the facet summary.  In `LuceneSearcher`:

https://github.com/geonetwork/core-geonetwork/blob/c0855205156ec0f8a6cb062071e9ac2afbd9a157/core/src/main/java/org/fao/geonet/kernel/search/LuceneSearcher.java#L1649-L1662

Probably can be closed, but the existing code is not even calling `IndexAndTaxonomy.close()`, but `_sm.releaseIndexReader(indexAndTaxonomy);` that does the same code as in `IndexAndTaxonomy.close()`:

https://github.com/geonetwork/core-geonetwork/blob/c0855205156ec0f8a6cb062071e9ac2afbd9a157/core/src/main/java/org/fao/geonet/kernel/search/SearchManager.java#L1211-L1213